File: weknot_android/app/src/main/java/com/example/weknot_android/network/api/VideoCallApi.java
package com.example.weknot_android.network.api;

import com.example.weknot_android.network.response.Response;
import com.example.weknot_android.network.response.data.LoginData;
import com.example.weknot_android.room.entity.videocall.VideoCall;

import io.reactivex.Single;
import retrofit2.http.Body;
import retrofit2.http.Header;
import retrofit2.http.POST;

public interface VideoCallApi {

    @POST("/requestCall")
    Single<retrofit2.Response<Response<VideoCall>>> requestCall(@Header("Authorization") String token);
}

File: weknot_android/app/src/main/java/com/example/weknot_android/network/api/VideoCallApi.kt
package com.example.weknot_android.network.api

import com.example.weknot_android.model.entity.user.Friend
import com.example.weknot_android.model.entity.videocall.VideoCall
import com.example.weknot_android.network.response.Response
import io.reactivex.Single
import retrofit2.http.*

interface VideoCallApi {

    @POST("/matching")
    fun requestCall(@Header("Authorization") token: String): Single<retrofit2.Response<Response<VideoCall>>>

    @POST("/matching/{channel}")
    fun finishRoom(@Header("Authorization") token: String,
                   @Path("channel") channel: String): Single<retrofit2.Response<Response<String>>>

    @POST("/matching/addPoint/{id}")
    fun addPoint(@Header("Authorization") token: String,
                 @Path("id") id: String): Single<retrofit2.Response<Response<Any>>>
}